ghost/staff-service/lib/email-templates/partials/commonStyles.hbs

```diff
@@ -0,0 +1,99 @@
+<style>
+  /* -------------------------------------
+      RESPONSIVE AND MOBILE FRIENDLY STYLES
+  ------------------------------------- */
+  @media only screen and (max-width: 620px) {
+    table[class=body] h1 {
+      font-size: 28px !important;
+      margin-bottom: 10px !important;
+    }
+    table[class=body] p,
+    table[class=body] ul,
+    table[class=body] ol,
+    table[class=body] td,
+    table[class=body] span,
+    table[class=body] a {
+      font-size: 16px !important;
+    }
+    table[class=body] .wrapper,
+    table[class=body] .article {
+      padding: 10px !important;
+    }
+    table[class=body] .content {
+      padding: 0 !important;
+    }
+    table[class=body] .container {
+      padding: 0 !important;
+      width: 100% !important;
+    }
+    table[class=body] .main {
+      border-left-width: 0 !important;
+      border-radius: 0 !important;
+      border-right-width: 0 !important;
+    }
+    table[class=body] .btn table {
+      width: 100% !important;
+    }
+    table[class=body] .btn a {
+      width: 100% !important;
+    }
+    table[class=body] .img-responsive {
+      height: auto !important;
+      max-width: 100% !important;
+      width: auto !important;
+    }
+    table[class=body] p[class=small],
+    table[class=body] a[class=small] {
+      font-size: 11px !important;
+    }
+  }
+  /* -------------------------------------
+      PRESERVE THESE STYLES IN THE HEAD
+  ------------------------------------- */
+  @media all {
+    .ExternalClass {
+      width: 100%;
+    }
+    .ExternalClass,
+    .ExternalClass p,
+    .ExternalClass span,
+    .ExternalClass font,
+    .ExternalClass td,
+    .ExternalClass div {
+      line-height: 100%;
+    }
+    /* Reset styles for Gmail (it wraps email address in link with custom styles) */
```

Comment:
The PR looks large, but in general I just added this comment and add `text-link` class to markup. All others  changes - just moving common styles to the separate file